### PR TITLE
Coverage/remove old rake tasks

### PR DESCRIPTION
**Pull request by Caleb-T-Owens**
## Status

* Current Status: Ready for (front-end / tech) review
* Review App link(s): *populate this with links to the relevant parts of the review app*

## Review progress:

~- [ ] Browser tested~
~- [ ] Front-end review completed~
- [x] Tech review completed

## What's changed?

I've removed a bunch of old and unused rake tasks as well as testing some of the parts of the app which I've recent knowledge of that had low coverage

### CODE REVIEW

File: spec/models/enrichment_groupings/term_spec.rb [hunk 1]
```diff
@@ -22,8 +22,62 @@
...
+        end
+      end
+
+      context 'when days till start is behind in the year' do
+        subject { create(:enrichment_groupings_term, term_start: 2.days.ago, term_end: 1.days.ago) }
+
+        it 'should return 364' do
+          expect(subject.days_till_term).to eq 367
+        end
+      end
```
Comment (dcorking) [inline spec/models/enrichment_groupings/term_spec.rb:80]: Will this break in a leap year? If so, use Timecop.freeze to run it on a fixed data.

**Review by dcorking — COMMENTED** [review]
other than the possible time-dependent test, this is great.

Importantly some of the rake tasks you deleted could have been harmful if they were run in production.

File: spec/models/enrichment_groupings/term_spec.rb (hunk 1 above)
Comment (Caleb-T-Owens) [reply spec/models/enrichment_groupings/term_spec.rb:80]: No it won't as we "normalize" the dates to the year 2000

**Review by dcorking — APPROVED** [review]
LGTM 🚂

**Pull request merged**